lib/src/ui/scaner_screen.dart

```diff
@@ -57,6 +57,8 @@ class ScanerScreenState extends State<ScanerScreen> {
             builder: (context, snapshot) => Column(
               mainAxisAlignment: MainAxisAlignment.end,
               children: [
+                if (bleScanner.state.discoveredDevices.isNotEmpty)
```

Comment:
We can remove `if`. Text can be visible always

Reply:
Agree

Reply:
Removed together with text widget (device list is going to be appBar part (as we discussed))